woocommerce/woocommerce
# Add post checkout pnpm install to git hooks

### All Submissions:






### Changes proposed in this Pull Request:

In a further small step to try help developers using the monorepo, this adds a post-checkout hook to run `pnpm install`. This means when you're swapping branches you won't forget to get the latest state of pnpm dependencies and I hope it might be a step towards avoiding pitfalls when working in the monorepo.






### How to test the changes in this Pull Request:

1. Fetch this branch locally: `git fetch origin dev/install-on-checkout`
2. Now checkout this branch: `git checkout dev/install-on-checkout` 
3. a PNPM install should run



### Other information:




### FOR PR REVIEWER ONLY:
